src/components/molecules/ContentSearchbar.vue

```diff
@@ -0,0 +1,140 @@
+<template>
+	<div class="search">
+		<div :class="[isActive ? 'input-active' : '', 'search__container']">
+			<input
+				ref="searchInput"
+				v-autowidth="{ maxWidth: '960px', minWidth: '0px', comfortZone: 40 }"
+				:value="value"
+				:aria-label="ariaLabel"
+				label="search-input"
+				type="text"
+				name="search"
+				v-bind="$attrs"
+				@keyup.enter="enterKeyHandler"
+				@input="updateSearchString($event.target.value)"
+				@focus="isActive = true"
+			/>
+			<div class="search__container--icon">
+				<base-icon
+					v-if="isActive"
+					class="search-icon"
+					source="custom"
+					icon="search"
+					@click="enterKeyHandler"
+				/>
+				<base-icon
+					v-else
+					class="search-icon"
+					source="custom"
+					icon="clear"
+					@click="clearBtnHandler"
+				/>
+			</div>
+		</div>
+	</div>
+</template>
+
+<script>
+export default {
+	props: {
+		value: {
+			type: String,
+			default: "",
+		},
+		ariaLabel: {
+			type: String,
+			default: "Search",
+		},
+		loading: {
+			type: Boolean,
+		},
+	},
+	data() {
+		return {
+			isActive: true,
+			inputValue: "",
+		};
+	},
+	mounted() {
+		window.addEventListener("keydown", this.backspaceKeyHandler);
+		this.$refs.searchInput.focus();
+	},
+	beforeDestroy() {
+		window.removeEventListener("keydown", this.backspaceKeyHandler);
+	},
+	methods: {
+		updateSearchString(newValue) {
+			this.inputValue = newValue;
+			this.$emit("input", this.inputValue);
+		},
+		enterKeyHandler() {
+			if (this.isActive && this.inputValue.length > 0) {
+				this.isActive = false;
+				this.inputValue = "";
+			}
+			this.$emit("enter-key-event");
```

Comment:
```suggestion
		enterKeyHandler(...args) {
			if (this.isActive && this.inputValue.length > 0) {
				this.isActive = false;
				this.inputValue = "";
			}
			this.$emit("keyup:enter", ...args);
```
the event name is not really declerative. From the current name it is not clear when this event will be triggered.
I also suggest passing the event itself through just in case the parent needs it.